fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -2280,6 +2360,10 @@ private static String getUrlPath(Bundle.Entry entry) {
                         } else if (request.getMethod().equals(HTTPVerb.PUT) && resource.getId() != null) {
                             // Add mapping.
                             addLocalRefMapping(localRefMap, localIdentifier, null, resource);
+                        } else if (request.getMethod().equals(HTTPVerb.PATCH) && resource.getId() != null) {
+                            // Add mapping.
+                            log.fine("Creating Patch Local Identefier For Bundle Patch Request");
+                            addLocalRefMapping(localRefMap, localIdentifier, null, resource);
```

Comment:
I don't think we should do any bundle-local reference rewriting in the case of patch requests, but I'm not really sure.
@prb112 / @JohnTimm / others: any thoughts?

Reply:
@Mani-Chandu-ihe do you think you need this for your use-case or were you just trying to copy what was done for PUT?  I assume the latter, but do let me know if I'm mistaken.

Reply:
> I don't think we should do any bundle-local reference rewriting in the case of patch requests, but I'm not really sure.
> @prb112 / @JohnTimm / others: any thoughts?

I agree - I think we should also put something in the conformance.md to explain it.

Reply:
Mani updated the PR to remove this code; I think its good now.  We *won't* do any local bundle reference rewriting for PATCH operations, but I think that is good.  If we find a use case that needs it, we can consider adding it then.